pages/index.js

```diff
@@ -1,7 +1,35 @@
+import ArtPiecePreview from "@/components/ArtPiecePreview/ArtPiecePreview";
+import ArtPieces from "@/components/ArtPieces/ArtPieces";
+import useSWR from "swr";
+
+const URL = "https://example-apis.vercel.app/api/art";
+
+async function fetcher(url) {
+  const res = await fetch(url);
+
+  if (!res) {
+    const error = new Error(`An error occured...`);
+    error.info = await res.json();
+    error.status = res.status;
+    throw error;
+  }
+
+  return res.json();
+}
+
 export default function HomePage() {
+  const { data, error, isLoading, isValidating, mutate } = useSWR(URL, fetcher);
```

Comment:
```suggestion
  const { data, error, isLoading } = useSWR(URL, fetcher);
```
When you don't use mutate and isValidating for now, you don't have to destructure it here.